### PR TITLE
i18n(pt-br): Update showcase.mdx

### PR DESCRIPTION
**Pull request by delucis**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- Updates the Portuguese translation of the Showcase page.
- Almost all of the content is the same, with mainly formatting changing.

### CODE REVIEW

File: docs/src/content/docs/pt-br/showcase.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-blog"
+		title="starlight-blog"
+		description="Adicione um blog ao seu site de documentação."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-links-validator"
+		title="starlight-links-validator"
+		description="Verifique se há links quebrados em suas páginas Starlight."
```
Comment (delucis) [inline docs/src/content/docs/pt-br/showcase.mdx:114]: Needs checking. Original English:

> Check for broken links in your Starlight pages.

File: docs/src/content/docs/pt-br/showcase.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-typedoc"
+		title="starlight-typedoc"
+		description="Gere páginas do Starlight a partir do TypeScript utilizando TypeDoc."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-openapi"
+		title="starlight-openapi"
+		description="Crie páginas de documentação a partir de especificações OpenAPI/Swagger."
```
Comment (delucis) [inline docs/src/content/docs/pt-br/showcase.mdx:124]: Needs checking. Original English:
> Create documentation pages from OpenAPI/Swagger specifications.

File: docs/src/content/docs/pt-br/showcase.mdx (hunk 1 above)
Comment (Alynva) [reply docs/src/content/docs/pt-br/showcase.mdx:114]: LGTM!

File: docs/src/content/docs/pt-br/showcase.mdx (hunk 2 above)
Comment (Alynva) [reply docs/src/content/docs/pt-br/showcase.mdx:124]: LGTM!

**Comment by delucis** [comment]
Thanks for reviewing @Alynva — appreciate it 💜

**Pull request merged**